#!/usr/bin/env python3

from dynamic_reconfigure.parameter_generator_catkin import *
PACKAGE = "backend"

gen = ParameterGenerator()

gen.add("speed_param",    double_t,    0,
        "Dynamic Parameter of the Robot speed", 300,  100, 500)

exit(gen.generate(PACKAGE, PACKAGE, "Speed"))